templates/home.html

```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
         <h2 class="home-description">THE NEXT GENERATION OF MUSIC</h2>
         {% if user.is_authenticated %}
             <div class="home-boxes">
-                <a class="button text-center box-margin" href={% url 'home' %}#how>I'm an artist</a>
-                <a class="button text-center box-margin" href={% url 'home' %}#fan>I'm a fan</a>
+                <a class="button text-center box-margin view-artists-box" href={% url 'artists' %}>START INVESTING</a>
             </div>
         {% else %}
             <div class="home-boxes">
-                <a class="button text-center box-margin" href="{% url 'register' %}?next={{ request.get_full_path }}">SIGN UP</a>
-                <a class="button text-center box-margin" data-open="login-modal">LOGIN</a>
+                <a class="button text-center box-margin view-artists-box" href={% url 'artists' %}>START INVESTING</a>
             </div>
         {% endif %}
         <div class="bottom-how">
             <h5><a class="white-color" href={% url 'home' %}#how>HOW IT WORKS<br/>
             <i class="fa fa-chevron-down fa-1x" aria-hidden="true"></i></a></h5>
         </div>
     </div>
-    <!-- How it works for fans -->
+        <!-- How it works for fans -->
     <div id="how-page" class="page">
         <a id="how" class="smooth"></a>
-        
-        <h3 class="how-title">HOW IT WORKS FOR FANS</h3>
+        <h3 class="extra-padding">WHAT IS PERDIEM?</h3>
+        <div class="video-padding">
+            <div class="videowrapper">
+                <iframe width="1280" height="720" src="https://www.youtube.com/embed/M78Ep_qcnpA" frameborder="0" allowfullscreen></iframe>
+            </div>
+        </div>
+        <h4 class="more-extra-padding">WE HELP ARTISTS GET FUNDING BY HAVING FANS INVEST IN THEIR MUSIC</h4>
+        <div class="break">
+            <hr>
+        </div>
+        <h3 class="how-title">HOW IT WORKS FOR ARTISTS</h3>
```

Comment:
I'd be curious to understand why you decided to reorder the artists section before fans. I think having fans first makes sense, since naturally the majority of users on the site will be fans.